datatypes/src/raster/raster_tile.rs

```diff
@@ -44,6 +45,24 @@ pub struct BaseTile<G> {
     pub grid_array: G,
     /// Metadata for the `BaseTile`
     pub properties: RasterProperties,
+    /// Indicate how long the tile may be cached, if `None` the tile may be cached indefinitely.
+    pub cache_hint: CacheHint,
+}
+
+// Manual implementation of equality because the cache hint field contains the creation time of the tile and shall not be considered for equality.
+// TODO: find a different solution for this(?), e.g. not inclduing the creation time, or by implementing PartialEq on CacheHint using only the expiration date.
+impl<G> PartialEq for BaseTile<G>
+where
+    G: PartialEq,
+{
+    fn eq(&self, other: &Self) -> bool {
+        self.time == other.time
+            && self.tile_position == other.tile_position
+            && self.global_geo_transform == other.global_geo_transform
+            && self.grid_array == other.grid_array
+            && self.properties == other.properties
+        // deliberately omitted: self.cache_hint == other.cache_hint
+    }
```

Comment:
1. We should implement stuff for production, not test.
So we rather impl a new comparison method that we can use there.

2. Now we should think how we will use it in production? Can the IDE find uses of that?
If so, we should make it so that it does the right thing there.

Reply:
> Now we should think how we will use it in production? Can the IDE find uses of that?

Unfortunately it can't. But I guess even if it wasn't used, you are right, it should return something useful in prod rather than for tests.

So should, in practice, two tiles with the same position, time and data but different creation and expiration times considered the same? I guess not, if they were the same, they could be used interchangeable without any implications and with the cache enabled this is not the case.

So I guess I should include the cache hint in the comparison and adjust the tests